.gitignore

```diff
@@ -0,0 +1,9 @@
+# General ignores
+node_modules/
+*.log
+# Environment files
+.env.local
+client/package.json
+client/package-lock.json
+package.json
```

Comment:
You should push the package.json and ignore the server package-lock.json as well